test/TorSharp.Tests/ARM/FetcherTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Net.Http;
+using System.Threading.Tasks;
+
+using Knapcode.TorSharp.Tools.Tor;
+
+using Xunit;
+
+namespace Knapcode.TorSharp.Tests.ARM
```

Comment:
We should have a test suite for the system-provided privoxy and tor. I wonder if we can do an apt-get install for privoxy and tor in the GitHub Actions YAML and then test the system-provided flow.

Reply:
I believe you can run any command inside. Docs says you can run apt install.
https://docs.github.com/en/actions/using-github-hosted-runners/customizing-github-hosted-runners